var/spack/repos/builtin/packages/py-pyqt4/package.py

```diff
@@ -25,19 +25,76 @@ class PyPyqt4(SIPPackage):
     ]
 
     version('4.12.3', sha256='a00f5abef240a7b5852b7924fa5fdf5174569525dc076cd368a566619e56d472')
-    version('4.11.3', '997c3e443165a89a559e0d96b061bf70',
+    version('4.11.3', sha256='853780dcdbe2e6ba785d703d059b096e1fc49369d3e8d41a060be874b8745686',
             url='http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.3/PyQt-x11-gpl-4.11.3.tar.gz')
 
+    variant('qsci', default=False, description='Build with QScintilla python bindings')
+
     # Supposedly can also be built with Qt 5 compatibility layer
     depends_on('qt@:4')
+    depends_on('qscintilla', when='+qsci')
```

Comment:
I think this is no longer necessary now that you are using a resource?

Reply:
It is necessary since qsci python bindings need to be linked with `-lqscintilla2_qt` as shown in make_qsci function. The only thing is we need to make sure qscintilla resource version is the same as qscintilla depenency. I will push a commit for it soon.

Reply:
Actually, I just realized that `when='^qscintilla@2.10.2'` statement in resource makes sure that resource version will be the same as that of dependency. So, no need to fix anything here I guess except it would nice if user got an error message right away as opposed to until `install` phase if no resource is defined for the version of qscintilla dependency.